src/test/java/de/azapps/kafkabackup/common/record/RecordJSONSerdeTest.java

```diff
@@ -2,120 +2,110 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.module.SimpleModule;
-import de.azapps.kafkabackup.common.record.Record;
 
 import static org.junit.Assert.assertEquals;
 
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.io.*;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.nio.file.Files;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
 import java.util.Base64;
-import java.util.Comparator;
 
 public class RecordJSONSerdeTest {
 
+    private static final String topic = "test-topic";
+    private static final int partition = 42;
+    private static final long offset = 123;
+    private static byte[] keyBytes;
+    private static String keyBase64;
+    private static byte[] valueBytes;
+    private static String valueBase64;
+    private static final String JSON_ENCODING = "UTF-8";
+    private static ObjectMapper mapper;
+    private static RecordJSONSerde serde;
+
+    @BeforeClass
+    public static void beforeAll() throws Exception {
+        // encoding here is not really important, we just want some bytes
+        keyBytes = "test-key".getBytes("UTF-8");
+        valueBytes = "test-value".getBytes("UTF-8");
+        // using Base64 as encoding in the json is part of our Record serialization format however:
+        keyBase64 = Base64.getEncoder().encodeToString(keyBytes);
+        valueBase64 = Base64.getEncoder().encodeToString(valueBytes);
+    }
+
+    @Before
+    public void beforeEach() throws Exception {
+        mapper = new ObjectMapper();
+        SimpleModule module = new SimpleModule();
+        module.addDeserializer(Record.class, new RecordJSONSerde.Deserializer());
+        module.addSerializer(Record.class, new RecordJSONSerde.Serializer());
+        mapper.registerModule(module);
+
+        serde = new RecordJSONSerde();
```

Comment:
small thing - have you ever heard about convention with naming variable `sut` (Service under test) - for me it's a good clarification what is tested in some specific class/scope

Reply:
Ah. That sounds nice. However, we have a bit of a weird mix here, since we are both testing the ObjectMapper and the RecordJSONSerde... 
`sutSerde` and `sutMapper`?

Reply:
hmm, so maybe it suggests that this class should be separated into two test suits? 🤔if no and we still want to have this in single file -  I'm okay with sutSerde and sutMapper 👍

Reply:
Fixed in a follow up PR